tasks/section5.yml

```diff
@@ -171,15 +272,20 @@
       - rule_5.2.1
 
 - name: "SCORED | 5.2.1 | PATCH | Ensure permissions on /etc/ssh/sshd_config are configured"
-  command: /bin/true
+  file:
+      dest: /etc/ssh/sshd_config
+      state: file
+      owner: root
+      group: root
+      mode: 0600
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.1
 
 - name: "SCORED | 5.2.2 | AUDIT | Ensure SSH Protocol is set to 2"
-  command: /bin/true
+  command: grep "^Protocol 2" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*